app/src/views/private/components/sidebar-detail.vue

```diff
@@ -19,27 +18,27 @@ const { active, toggle } = useGroupable({
 	group: 'sidebar-detail',
 });
 
-const appStore = useAppStore();
-const { sidebarOpen } = toRefs(appStore);
+const sidebarStore = useSidebarStore();
 
 function onClick() {
 	emit('toggle', !active.value);
+	sidebarStore.expand();
 	toggle();
 }
```

Comment:
call toggle before expand so that the value isn't automatically set back to the first item in the sidebar
```suggestion
function onClick() {
	emit('toggle', !active.value);
	toggle();
	sidebarStore.expand();
}
```